packages/ui-extensions-server-kit/src/styles/theme.module.scss

```diff
@@ -128,6 +128,7 @@
   --p-decorative-five-icon: rgba(174, 43, 76, 1);
   --p-decorative-five-surface: rgba(253, 201, 208, 1);
   --p-decorative-five-text: rgba(79, 14, 31, 1);
+  --p-status-border-radius: 10px;
```

Comment:
NABD... should we standardize this to use `rem` instead of `px`?

Reply:
preferrably yes. We can also do this in the clean up ticket, our things are all over the place atm.